### ISSUE TITLE
Error with TS 3.3.1

### ISSUE THREAD

**Issue by willdurand** [typescript-eslint/typescript-eslint#181]
I know it is not officially supported yet and you are probably aware of the problem but I did not find any issue for that here, so I filed this one to be sure.

With TS 3.3.1, it looks like the plugin does not find `tsutils` anymore:

```
$ eslint --ext ts --ext js --ext tsx src/
Cannot read config file: /home/travis/build/mozilla/addons-code-manager/node_modules/@typescript-eslint/eslint-plugin/lib/index.js
Error: Cannot find module 'tsutils'
Referenced from: /home/travis/build/mozilla/addons-code-manager/.eslintrc
Error: Cannot read config file: /home/travis/build/mozilla/addons-code-manager/node_modules/@typescript-eslint/eslint-plugin/lib/index.js
Error: Cannot find module 'tsutils'
Referenced from: /home/travis/build/mozilla/addons-code-manager/.eslintrc
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/travis/build/mozilla/addons-code-manager/node_modules/@typescript-eslint/eslint-plugin/lib/rules/no-unnecessary-type-assertion.js:7:17)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

See this Travis job for full output: https://travis-ci.com/mozilla/addons-code-manager/jobs/174650457

**Comment by armano2** [comment]
this is not issue with typescript, its issue with missing dependencies on production build in project.

this issue got solved in #165 and it's available in canary release

**Comment by JamesHenry** [comment]
I was literally just writing at the same time, an update though - it is available in the latest _stable_ release 1.2.0.
